File: appjedrez/src/app/models/notificacion.ts
import { Torneo } from './torneo';
import { Usuario } from './usuario';

export interface Notificacion {
  id: string;
  usuarioId: string;
  torneoId: string;
  mensaje: string;
  estado: string;
  fecha: string;
  usuario: Usuario;
  torneo: Torneo;
}

File: AppjedrezApi/AuthApi/Models/Notificacion.cs
using System;
using System.Collections.Generic;

namespace AuthApi.Models;

public partial class Notificacion
{
    public Guid Id { get; set; }

    public Guid? UsuarioId { get; set; }

    public string Mensaje { get; set; }

    public string Estado { get; set; }

    public DateTime? Fecha { get; set; }

    public Guid? TorneoId { get; set; }

    public virtual Torneo Torneo { get; set; }

    public virtual Usuario Usuario { get; set; }
}
